docs/topics/components-stability.md

```diff
@@ -59,37 +59,38 @@ We make sure to document precisely which subcomponents are not stable. We also d
+| Kotlin K2 (JVM)                                       | Alpha        | 1.7                      |                                                                                   |
+| kotlin-stdlib (JVM)                                   | Stable       | 1.0                      |                                                                                   |
+| Coroutines                                            | Stable       | 1.3                      |                                                                                   |
+| kotlin-reflect (JVM)                                  | Beta         | 1.0                      |                                                                                   |
+| Kotlin/JS (Classic back-end)                          | Stable       | 1.3                      | Deprecated from 1.8.0, read the [IR migration guide](js-ir-migration.md)          |
+| Kotlin/JVM (IR-based)                                 | Stable       | 1.5                      |                                                                                   |
+| Kotlin/JS (IR-based)                                  | Stable       | 1.8                      |                                                                                   |
+| Kotlin/Native Runtime                                 | Stable       | 1.9.20                   |                                                                                   |
+| Kotlin/Native memory manager                          | Stable       | 1.9.20                   |                                                                                   |
+| klib binaries                                         | Stable       | 1.9.20                   |                                                                                   |
+| Kotlin Multiplatform                                  | Stable       | 1.9.20                   |                                                                                   |
+| Kotlin/Native interop with C and Objective C          | Beta         | 1.3                      |                                                                                   |
+| CocoaPods integration                                 | Stable       | 1.9.20                   |                                                                                   |
+| Kotlin Multiplatform Mobile plugin for Android Studio | Beta         | 0.8.0                    | [Versioned separately from the language](multiplatform-mobile-plugin-releases.md) |
+| expected and actual functions and properties          | Stable       | 1.9.20                   |                                                                                   |
+| expected and actual classes                           | Beta         | 1.2                      |                                                                                   |
```

Comment:
Hm... KMM mobile went Beta in 1.7.20... I think we should have 1.7.20 here, not 1.2. (And previous 1.2 version here was a mistake -- it was experimental since 1.2, then at some point Alpha and in 1.7.20 -- Beta